lib/chainer/functions/array/broadcast_to.rb

```diff
@@ -0,0 +1,43 @@
+module Chainer
+  module Functions
+    module Array
+      # Function that broadcasts an array to a new shape."""
+      class BroadcastTo < Function
+        def initialize(shape)
+            @shape = shape
+        end
+
+        def self.broadcast_to(xs, shape)
+          self.new(shape).(xs)
+        end
+
+        def forward_cpu(xs)
```

Comment:
Here, you define the forward_cpu(), backward_cpu() method.
If there is no plans to separate the code between the CPU and the GPU in the future, I think that the forward(), backward() names are better as in the original chainer/functions/array/broadcast.py.

Reply:
@naitoh fix it.